GEON-PPANG-iOS/Presentation/BakeryDetail/ViewControllers/BakeryDetailViewController.swift

```diff
@@ -22,23 +22,25 @@ final class BakeryDetailViewController: BaseViewController {
                                                                                 isNonGMO: false,
                                                                                 firstNearStation: "",
                                                                                 secondNearStation: "",
-                                                                                isBookMarked: false,
-                                                                                bookMarkCount: 0,
                                                                                 reviewCount: 0,
                                                                                 breadType: BreadResponseType(breadTypeID: 0,
                                                                                                              name: "",
                                                                                                              isGlutenFree: false,
                                                                                                              isVegan: false,
                                                                                                              isNutFree: false,
                                                                                                              isSugarFree: false),
-                                                                                homepage: "",
+                                                                                mapURL: "",
+                                                                                homepageURL: "",
+                                                                                instagramURL: "",
                                                                                 address: "",
-                                                                                openingTime: "",
+                                                                                openingHours: "",
                                                                                 closedDay: "",
                                                                                 phoneNumber: "",
                                                                                 menuList: [MenuList(menuID: 0,
                                                                                                     menuName: "",
-                                                                                                    menuPrice: 0)]) {
+                                                                                                    menuPrice: 0)],
+                                                                                isBookMarked: false,
+                                                                                bookMarkCount: 0) {
```

Comment:
p4: BakeryDetailResponseDTO에 초기화함수 만드는건 어떠신지요 ? reviewData도 비슷하게 ,,, 
```suggestion
extention BakeryDetailResponseDTO {
static func initialDTO() -> BakeryDetailResponseDTO {
        return BakeryDetailResponseDTO(bakeryID: 0, bakeryName: "", bakeryPicture: "", isHACCP: false, isVegan: false, isNonGMO: false, firstNearStation: "", secondNearStation: "", isBookMarked: false, bookMarkCount: 0, reviewCount: 0, breadType: BreadResponseType(breadTypeID: 0, name: "", isGlutenFree: false, isVegan: false, isNutFree: false, isSugarFree: false), menuList: [MenuList(menuID: 0, menuName: "", menuPrice: 0)], mapURL: "", homepageURL: "", instagramURL: "", address: "", openingHours: "", closedDay: "", phoneNumber: "")
    }
}
private var overviewData: BakeryDetailResponseDTO = .initialDTO()
```

Reply:
reviewData는 리뷰 셀 디자인 수정할 때 같이 바꾸겠습니다 ~

[59653b2](https://github.com/GEON-PPANG/GEON-PPANG-iOS/pull/166/commits/59653b2eb35a787052129c8dffb05c7fba36bb43)